seedbuilder/areas.ori

```diff
@@ -1665,53 +1673,68 @@ home: BashTree 521 326
 	conn: BashTreeDoorClosed
 		casual Open
 	conn: UpperGinsoRedirectArea
-		casual Bash
-		casual ChargeJump Dash DoubleJump
+		casual Bash WallJump
+		casual Bash Climb
+		casual Bash DoubleJump
+		casual Bash ChargeJump
+		casual Bash Grenade
 		casual ChargeJump DoubleJump Glide
+		standard ChargeJump Dash DoubleJump
 		standard ChargeJump Health=4
+		standard Lure Bash
 		expert ChargeJump Health=3
-		expert ChargeJump ChargeDash
+		expert ChargeJump ChargeDash Energy=1
 		master DoubleJump Dash
 		master DoubleJump WallJump
 		master DoubleJump Climb
 		master TripleJump
-home: UpperGinsoRedirectArea 526 385
+home: UpperGinsoRedirectArea 521 388
 	-- located at the first redirect area on top of where the UpperGinsoRedirectLowerExp is found
 	-- for KS-lock protection, all pickups in this area require the ginso key
 	pickup: UpperGinsoRedirectLowerExp
 		casual GinsoKey Stomp
 		casual GinsoKey Bash
-		standard GinsoKey ChargeJump Climb
+		casual GinsoKey ChargeJump Climb
 		expert GinsoKey ChargeFlame
 		expert GinsoKey ChargeJump AirDash
 		master GinsoKey Grenade
 	pickup: UpperGinsoRedirectUpperExp
-		casual GinsoKey Stomp
 		casual GinsoKey Bash
```

Comment:
```suggestion
		standard GinsoKey Bash
```
getting to the upper section with just Bash requires to jump over the shot before bashing it to get enough height, then requires a precise bash angle to break the second pickup floor.